infra/feast-operator/config/samples/v1alpha1_featurestore_all_kubernetes_non_tls.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: feast.dev/v1alpha1
+kind: FeatureStore
+metadata:
+  name: sample-kubernetes-auth-non-tls
+spec:
+  feastProject: feast_rbac
+  authz:
+    kubernetes:
+      roles:
+        - feast-writer
+        - feast-reader
+  services:
+    offlineStore:
+      server:
+        tls:
+          disable: true
+    onlineStore:
+      server:
+        tls:
+          disable: true
+    registry:
+      local:
+        server:
+          tls:
+            disable: true
+    ui: {}
```

Comment:
this file should be removed... it is unnecessary. in a kubernetes cluster TLS is not default, only in openshift.
the same `FeatureStore` CR can be used in k8s or openshift... but in openshift TLS will be enabled by default

Reply:
you don't need a different CR file for k8s vs openshift

Reply:
Resolved!